src/api/c++/z3++.h

```diff
@@ -2935,7 +3079,26 @@ namespace z3 {
         Z3_fixedpoint m_fp;
     public:
         fixedpoint(context& c):object(c) { m_fp = Z3_mk_fixedpoint(c); Z3_fixedpoint_inc_ref(c, m_fp); }
-        ~fixedpoint() { Z3_fixedpoint_dec_ref(ctx(), m_fp); }
+		fixedpoint(fixedpoint const & o) : object{o}, m_fp{o.m_fp} {
+        	Z3_fixedpoint_inc_ref(ctx(), m_fp);
+		}
+		fixedpoint(fixedpoint && o) noexcept : object{std::forward<object>(o)}, m_fp{o.m_fp} {
+			o.m_fp = nullptr;
+		}
+		fixedpoint & operator=(fixedpoint const & o) {
```

Comment:
These were implicitly generated before and did not do reference counting as a result. I assume that was a bug.